web/app/js/components/MetricsTable.jsx

```diff
@@ -83,11 +83,16 @@ const columnDefinitions = (resource, namespaces, onFilterClick, showNamespaceCol
         } else {
           if (showGrafanaLink) {
             nameContents = (
-              <GrafanaLink
-                name={row.name}
-                namespace={row.namespace}
-                resource={resource}
-                PrefixedLink={PrefixedLink} />
+              <React.Fragment>
+                <PrefixedLink to={"/namespaces/" + row.namespace + "/" + resource + "s/" + row.name}>
+                  {row.name}
+                </PrefixedLink>
+                <GrafanaLink
+                  name={row.name}
+                  namespace={row.namespace}
+                  resource={resource}
+                  PrefixedLink={PrefixedLink} />
```

Comment:
It feels a little bit too subtle to me that the text links to one page and the icon links to a different site altogether, especially when we display them in line with each other:

![image](https://user-images.githubusercontent.com/9226/44281020-53426400-a20b-11e8-957d-ab7a1a8ea112.png)

I think it would be cool to put all of the grafana icons in their own column. Maybe we could add a "DASH" column as the last column in each table, and add the icon there for resources that have a corresponding dashboard?

Reply:
sure! that's fine by me, and  I think that's how we had it back in the day when we originally introduced the grafana links!

Reply:
![screen shot 2018-08-17 at 1 56 26 pm](https://user-images.githubusercontent.com/549258/44288542-8e519100-a225-11e8-9026-578c3f1ce8a9.png)